.gitignore

```diff
@@ -139,24 +132,25 @@ windows/
 
 napari/_version.py
 
-docs/api/napari*
-docs/_build
+napari/settings/napari.schema.json
 
 # built in setup.py
 napari/view_layers.pyi
 napari/components/viewer_model.pyi
 
-# Autogenerated documentation
-docs/images/_autogenerated/
+# Sphinx documentation
+docs/_build
+docs/_tags/
+docs/api/napari*
+docs/gallery/
 docs/guides/preferences.md
 docs/guides/_layer_events.md
-docs/guides/_viewer_events.md
 docs/guides/_layerlist_events.md
+docs/guides/_viewer_events.md
+docs/images/_autogenerated/
+docs/jupyter_execute/
+docs/sg_execution_times.rst
+
 # come from npe2 docs
 docs/plugins/_npe2_*.md
-
-napari/settings/napari.schema.json
-
-docs/jupyter_execute/
-docs/.jupyter_cache/
```

Comment:
Was this not needed anymore?

Reply:
You are right, the jupyter_execute still needs to be there.